AI/StupidAI/StupidAI.cpp

```diff
@@ -49,7 +49,8 @@ struct EnemyInfo
 	const CStack * s;
 	int adi, adr;
 	std::vector<BattleHex> attackFrom; //for melee fight
-	EnemyInfo(const CStack * _s) : s(_s), adi(0), adr(0)
+	EnemyInfo(const CStack * _s)
+		: s(_s), adi(0), adr(0)

```

Comment:
Any advantages of this formatting change except saving some space for really many parameters? For me it's little bit easier to track code when { } belong to something that is written in one line above {, but maybe I freak too much because I saw too many ugly multi-line if's and elses